teamcity: failed tests on master: testrace/TestParallel/subquery_retry, testrace/TestParallel

The following tests appear to have failed:

[#369677](https://teamcity.cockroachdb.com/viewLog.html?buildId=369677):

```
--- FAIL: testrace/TestParallel/subquery_retry (0.000s)
Test ended in panic.

------- Stdout: -------
panic: test timed out after 15m0s

goroutine 519651 [running]:
testing.startAlarm.func1()
	/usr/local/go/src/testing/testing.go:1145 +0x139
created by time.goFunc
	/usr/local/go/src/time/sleep.go:170 +0x52

goroutine 1 [chan receive]:
testing.(*T).Run(0xc42018c3c0, 0x2369415, 0xc, 0x24300a8, 0xc420647c01)
	/usr/local/go/src/testing/testing.go:790 +0x59b
testing.runTests.func1(0xc42018c3c0)
	/usr/local/go/src/testing/testing.go:1004 +0xa8
testing.tRunner(0xc42018c3c0, 0xc420647d50)
	/usr/local/go/src/testing/testing.go:746 +0x16d
testing.runTests(0xc42026e300, 0x33657a0, 0x2, 0x2, 0xc4204ce4e0)
	/usr/local/go/src/testing/testing.go:1002 +0x522
testing.(*M).Run(0xc420e1bf20, 0x18)
	/usr/local/go/src/testing/testing.go:921 +0x207
github.com/cockroachdb/cockroach/pkg/sql/logictest.TestMain(0xc420647f20)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/logictest/main_test.go:63 +0x15e
main.main()
	github.com/cockroachdb/cockroach/pkg/sql/logictest/_test/_testmain.go:44 +0x1ce

goroutine 34 [chan receive]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).flushDaemon(0x3859f80)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1005 +0x8f
created by github.com/cockroachdb/cockroach/pkg/util/log.init.0
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:581 +0x13a

goroutine 35 [syscall, 15 minutes]:
os/signal.signal_recv(0x7099b1)
	/usr/local/go/src/runtime/sigqueue.go:131 +0xa6
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:22 +0x30
created by os/signal.init.0
	/usr/local/go/src/os/signal/signal_unix.go:28 +0x4f

goroutine 519671 [chan receive]:
testing.(*T).Run(0xc430b0cc30, 0xc4234e3e47, 0xe, 0xc4233d5fc0, 0x2)
	/usr/local/go/src/testing/testing.go:790 +0x59b
github.com/cockroachdb/cockroach/pkg/sql/logictest.TestParallel(0xc430b0cc30)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/logictest/parallel_test.go:256 +0x256
testing.tRunner(0xc430b0cc30, 0x24300a8)
	/usr/local/go/src/testing/testing.go:746 +0x16d
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:789 +0x569

goroutine 519688 [select]:
github.com/cockroachdb/cockroach/pkg/gossip.(*Gossip).bootstrap.func1(0x339f920, 0xc42566ab40)
	/go/src/github.com/cockroachdb/cockroach/pkg/gossip/gossip.go:1136 +0x499
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c5c4db0, 0xc42228d4d0, 0xc42c5c4da0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519736 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0xfc00000000)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245aba40)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245aba40)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190a00, 0xc42b759050, 0xc42c1909f0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 128 [select, locked to thread]:
runtime.gopark(0x2432d10, 0x0, 0x2362915, 0x6, 0x18, 0x1)
	/usr/local/go/src/runtime/proc.go:277 +0xfd
runtime.selectgo(0xc420942f50, 0xc420355200)
	/usr/local/go/src/runtime/select.go:395 +0x12c3
runtime.ensureSigM.func1()
	/usr/local/go/src/runtime/signal_unix.go:511 +0x1a7
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2337 +0x1

goroutine 519737 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x15400000100)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245aba70)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245aba70)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190a20, 0xc42b759050, 0xc42c190a10)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519720 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x1b500000100)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245ab740)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245ab740)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190800, 0xc42b759050, 0xc42c1907f0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519751 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x9d00000000)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245abd10)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245abd10)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190c50, 0xc42b759050, 0xc42c190c40)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519628 [select]:
github.com/cockroachdb/cockroach/pkg/kv.(*TxnCoordSender).printStatsLoop(0xc420cfe1c0, 0x339f920, 0xc425077890)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/txn_coord_sender.go:253 +0x24e
github.com/cockroachdb/cockroach/pkg/kv.NewTxnCoordSender.func1(0x339f920, 0xc425077890)
	/go/src/github.com/cockroachdb/cockroach/pkg/kv/txn_coord_sender.go:237 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c5c4820, 0xc42228d4d0, 0xc42c5c4810)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519741 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x15d00000100)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245abb30)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245abb30)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190aa0, 0xc42b759050, 0xc42c190a90)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519685 [chan receive]:
github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux.muxListener.Accept(...)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux/cmux.go:184
github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux.(*muxListener).Accept(0xc423e80280, 0x2431fc0, 0xc424c55e00, 0x339eae0, 0xc423e80280)
	<autogenerated>:1 +0x8a
github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc.(*Server).Serve(0xc424c55e00, 0x339eae0, 0xc423e80280, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/vendor/google.golang.org/grpc/server.go:444 +0x217
github.com/cockroachdb/cockroach/pkg/server.(*Server).Start.func7(0x339f920, 0xc42566aa80)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:691 +0x8d
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c5c4d30, 0xc42228d4d0, 0xc423e803a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519715 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x16500000100)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245ab650)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245ab650)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190760, 0xc42b759050, 0xc42c190750)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519723 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x19d00000100)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245ab7d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245ab7d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190860, 0xc42b759050, 0xc42c190850)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519725 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x15500000100)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245ab830)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245ab830)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c1908a0, 0xc42b759050, 0xc42c190890)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519716 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x1a500000100)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245ab680)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245ab680)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190780, 0xc42b759050, 0xc42c190770)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519730 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x15d00000100)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245ab920)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245ab920)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190940, 0xc42b759050, 0xc42c190930)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519699 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0xb500000000)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc42459c0f0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc42459c0f0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190560, 0xc42b759050, 0xc42c190540)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519713 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x16d00000100)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245ab5c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245ab5c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190720, 0xc42b759050, 0xc42c190710)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519722 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x18500000100)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245ab7a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245ab7a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190840, 0xc42b759050, 0xc42c190830)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519752 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x11c00000100)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245abd40)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245abd40)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190c70, 0xc42b759050, 0xc42c190c60)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519750 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x7100000000)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245abce0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245abce0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190c30, 0xc42b759050, 0xc42c190bf0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519711 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x17500000100)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245ab560)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245ab560)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c1906e0, 0xc42b759050, 0xc42c1906d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519747 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x5300000000)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245abc50)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245abc50)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190b80, 0xc42b759050, 0xc42c190b70)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519690 [semacquire]:
sync.runtime_notifyListWait(0xc42b912af0, 0x5)
	/usr/local/go/src/runtime/sema.go:507 +0x11f
sync.(*Cond).Wait(0xc42b912ae0)
	/usr/local/go/src/sync/cond.go:56 +0x8e
github.com/cockroachdb/cockroach/pkg/storage/engine.(*RocksDB).syncLoop(0xc42b912a00)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/engine/rocksdb.go:478 +0xb3
created by github.com/cockroachdb/cockroach/pkg/storage/engine.(*RocksDB).open
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/engine/rocksdb.go:465 +0x656

goroutine 519683 [chan receive]:
github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux.muxListener.Accept(...)
	/go/src/github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux/cmux.go:184
github.com/cockroachdb/cockroach/vendor/github.com/cockroachdb/cmux.(*muxListener).Accept(0xc423e80320, 0x30, 0x22931e0, 0x1000000006d3b01, 0xc42566aa50)
	<autogenerated>:1 +0x8a
crypto/tls.(*listener).Accept(0xc423e80360, 0xc420086060, 0x2167b60, 0x33602a0, 0x23347c0)
	/usr/local/go/src/crypto/tls/tls.go:52 +0x56
net/http.(*Server).Serve(0xc423a94820, 0x3392a60, 0xc423e80360, 0x0, 0x0)
	/usr/local/go/src/net/http/server.go:2695 +0x203
github.com/cockroachdb/cockroach/pkg/server.(*Server).Start.func5(0x339f920, 0xc42566a9f0)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:675 +0x8f
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c5c4d10, 0xc42228d4d0, 0xc423e80380)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519724 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0xbd00000000)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245ab800)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245ab800)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190880, 0xc42b759050, 0xc42c190870)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519729 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x18d00000100)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245ab8f0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245ab8f0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190920, 0xc42b759050, 0xc42c190910)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519719 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x1bd00000100)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245ab710)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245ab710)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c1907e0, 0xc42b759050, 0xc42c1907d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519738 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x12400000100)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245abaa0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245abaa0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190a40, 0xc42b759050, 0xc42c190a30)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519726 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0xf400000000)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245ab860)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245ab860)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c1908c0, 0xc42b759050, 0xc42c1908b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519684 [chan receive]:
github.com/cockroachdb/cockroach/pkg/server.(*Server).Start.func6(0x339f920, 0xc424fc88d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:679 +0xc1
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c5c4d20, 0xc42228d4d0, 0xc421052cc0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519739 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0xcd00000000)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245abad0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245abad0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190a60, 0xc42b759050, 0xc42c190a50)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519687 [chan receive]:
github.com/cockroachdb/cockroach/pkg/gossip.(*server).start.func3(0x339f920, 0xc425077b30)
	/go/src/github.com/cockroachdb/cockroach/pkg/gossip/server.go:384 +0x83
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c5c4d90, 0xc42228d4d0, 0xc423e80460)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519734 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0xdd00000000)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245ab9e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245ab9e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c1909c0, 0xc42b759050, 0xc42c1909b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519703 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x7b00000000)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245ab3b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245ab3b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c1905e0, 0xc42b759050, 0xc42c1905d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519714 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x14c00000100)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245ab620)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245ab620)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190740, 0xc42b759050, 0xc42c190730)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519705 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0xc500000000)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245ab410)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245ab410)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190620, 0xc42b759050, 0xc42c190610)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519728 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x14400000100)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245ab8c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245ab8c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190900, 0xc42b759050, 0xc42c1908f0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519672 [semacquire]:
sync.runtime_Semacquire(0xc42b75907c)
	/usr/local/go/src/runtime/sema.go:56 +0x39
sync.(*WaitGroup).Wait(0xc42b759070)
	/usr/local/go/src/sync/waitgroup.go:131 +0xb4
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc42b759050, 0x339f920, 0xc42393d6b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:423 +0x2a2
github.com/cockroachdb/cockroach/pkg/server.bootstrapCluster(0x339f920, 0xc42393d6b0, 0x339b7e0, 0xc426890420, 0x0, 0xc421356c80, 0x339f920, 0xc42393d6b0, 0xbebc200, 0xf, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:245 +0xf7f
github.com/cockroachdb/cockroach/pkg/server.(*Node).bootstrap(0xc42428a900, 0x339f920, 0xc42393d6b0, 0xc42025bf80, 0x1, 0x1, 0x100000001, 0x200000000, 0x100000001, 0x200000000, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/node.go:342 +0x1dd
github.com/cockroachdb/cockroach/pkg/server.(*Server).Start(0xc42176d900, 0x339f920, 0xc42393d6b0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:833 +0x5d56
github.com/cockroachdb/cockroach/pkg/server.(*TestServer).Start(0xc42d69f140, 0x0, 0x0, 0x0, 0x0, 0x33844e0, 0xc4258b2300, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/testserver.go:323 +0x25d
github.com/cockroachdb/cockroach/pkg/testutils/serverutils.StartServerRaw(0x0, 0x0, 0x0, 0x0, 0x33844e0, 0xc4258b2300, 0x0, 0x0, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/testutils/serverutils/test_server_shim.go:197 +0x12a
github.com/cockroachdb/cockroach/pkg/testutils/serverutils.StartServer(0x33af620, 0xc4258b21c0, 0x0, 0x0, 0x0, 0x0, 0x33844e0, 0xc4258b2300, 0x0, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/testutils/serverutils/test_server_shim.go:167 +0x66
github.com/cockroachdb/cockroach/pkg/testutils/testcluster.(*TestCluster).doAddServer(0xc420191570, 0x33af620, 0xc4258b21c0, 0x0, 0x0, 0x0, 0x0, 0x33844e0, 0xc4258b2300, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/testutils/testcluster/testcluster.go:248 +0x154
github.com/cockroachdb/cockroach/pkg/testutils/testcluster.StartTestCluster(0x33af620, 0xc4258b21c0, 0x1, 0x0, 0x0, 0x0, 0x0, 0x33844e0, 0xc4258b2300, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/testutils/testcluster/testcluster.go:156 +0x68b
github.com/cockroachdb/cockroach/pkg/testutils/testcluster.testClusterFactoryImpl.StartTestCluster(0x33af620, 0xc4258b21c0, 0x1, 0x0, 0x0, 0x0, 0x0, 0x33844e0, 0xc4258b2300, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/testutils/testcluster/testcluster.go:553 +0x8e
github.com/cockroachdb/cockroach/pkg/testutils/testcluster.(*testClusterFactoryImpl).StartTestCluster(0x421f3c0, 0x33af620, 0xc4258b21c0, 0x1, 0x0, 0x0, 0x0, 0x0, 0x33844e0, 0xc4258b2300, ...)
	<autogenerated>:1 +0xc8
github.com/cockroachdb/cockroach/pkg/testutils/serverutils.StartTestCluster(0x33af620, 0xc4258b21c0, 0x1, 0x0, 0x0, 0x0, 0x0, 0x33844e0, 0xc4258b2300, 0x0, ...)
	/go/src/github.com/cockroachdb/cockroach/pkg/testutils/serverutils/test_cluster_shim.go:120 +0xd5
github.com/cockroachdb/cockroach/pkg/sql/logictest.(*parallelTest).setup(0xc4258b21c0, 0xc4258b2200)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/logictest/parallel_test.go:197 +0x2da
github.com/cockroachdb/cockroach/pkg/sql/logictest.(*parallelTest).run(0xc4258b21c0, 0xc4234e3e30, 0x25)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/logictest/parallel_test.go:151 +0x76f
github.com/cockroachdb/cockroach/pkg/sql/logictest.TestParallel.func1(0xc430b0ce10)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/logictest/parallel_test.go:258 +0x149
testing.tRunner(0xc430b0ce10, 0xc4233d5fc0)
	/usr/local/go/src/testing/testing.go:746 +0x16d
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:789 +0x569

goroutine 519763 [semacquire]:
sync.runtime_Semacquire(0xc42b759164)
	/usr/local/go/src/runtime/sema.go:56 +0x39
sync.(*WaitGroup).Wait(0xc42b759158)
	/usr/local/go/src/sync/waitgroup.go:131 +0xb4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Wait(0xc42b7590e0, 0x339f920, 0xc4245abf50)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:173 +0x3f
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Wait-fm(0x339f920, 0xc4245abf50)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/store.go:3640 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190e00, 0xc42b759050, 0xc42c190df0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519627 [chan receive]:
github.com/cockroachdb/cockroach/pkg/rpc.NewContext.func1(0x339f920, 0xc42566a990)
	/go/src/github.com/cockroachdb/cockroach/pkg/rpc/context.go:216 +0x6c
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c5c4520, 0xc42228d4d0, 0xc42c589be0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519753 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x11400000100)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245abd70)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245abd70)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190c90, 0xc42b759050, 0xc42c190c80)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519748 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x6300000000)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245abc80)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245abc80)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190bb0, 0xc42b759050, 0xc42c190ba0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519717 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x19500000100)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245ab6b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245ab6b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c1907a0, 0xc42b759050, 0xc42c190790)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519745 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x6900000000)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245abbf0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245abbf0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190b30, 0xc42b759050, 0xc42c190b10)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519759 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x5b00000000)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245abe90)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245abe90)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190d80, 0xc42b759050, 0xc42c190d70)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519727 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x10400000000)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245ab890)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245ab890)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c1908e0, 0xc42b759050, 0xc42c1908d0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519762 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0xe500000000)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245abf20)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245abf20)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190de0, 0xc42b759050, 0xc42c190dd0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519733 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x17d00000100)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245ab9b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245ab9b0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c1909a0, 0xc42b759050, 0xc42c190990)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519632 [chan receive]:
github.com/cockroachdb/cockroach/pkg/server.(*Server).Start.func2(0x339f920, 0xc42566a9c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/server/server.go:644 +0x84
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c5c4ce0, 0xc42228d4d0, 0xc423e802e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519708 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0x10c00000100)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245ab4a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).Start.func2(0x339f920, 0xc4245ab4a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:167 +0x56
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc42c190680, 0xc42b759050, 0xc42c190670)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:198 +0x161
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:191 +0xbb

goroutine 519732 [semacquire]:
sync.runtime_SemacquireMutex(0xc42b7590fc, 0xa500000000)
	/usr/local/go/src/runtime/sema.go:71 +0x3d
sync.(*Mutex).Lock(0xc42b7590f8)
	/usr/local/go/src/sync/mutex.go:134 +0x14c
github.com/cockroachdb/cockroach/pkg/util/syncutil.(*Mutex).Lock(0xc42b7590f8)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/syncutil/mutex_sync.go:32 +0x3b
sync.(*Cond).Wait(0xc429aa4f00)
	/usr/local/go/src/sync/cond.go:57 +0xa4
github.com/cockroachdb/cockroach/pkg/storage.(*raftScheduler).worker(0xc42b7590e0, 0x339f920, 0xc4245ab980)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/scheduler.go:198 +0x9c
github.com/cock
```
Please assign, take a look and update the issue accordingly.

--- a/Makefile
+++ b/Makefile
@@ -28,7 +28,7 @@ TESTS        :=.## Tests to run for use with `make test`.
 BENCHES      :=## Benchmarks to run for use with `make bench`.
 FILES        :=## Space delimited list of logic test files to run, for make testlogic.
 TESTTIMEOUT  := 4m## Test timeout to use for regular tests.
-RACETIMEOUT  := 15m## Test timeout to use for race tests.
+RACETIMEOUT  := 25m## Test timeout to use for race tests.
 ACCEPTANCETIMEOUT := 30m## Test timeout to use for acceptance tests.
 BENCHTIMEOUT := 5m## Test timeout to use for benchmarks.
 TESTFLAGS    :=## Extra flags to pass to the go test runner, e.g. "-v --vmodule=raft=1"

